cognite_toolkit/modules/cdf_data_pipeline_asset_default/auth/asset.read.groups.yaml

```diff
@@ -0,0 +1,18 @@
+# This role is identical to the normal users, but is used to group user with additional InField feature capabilities for checklist admins.
```

Comment:
Do you mean here that this group should get additional capabilities or that the infield checklist/template admin group's sourceId should match {{asset_location_read_group_source_id}}?